e2e/tests/tests/eigen_kms.rs

```diff
@@ -49,10 +51,17 @@ fn verify_signature_recovery(
 ) -> Result<()> {
     let secp = Secp256k1::new();
     let recovered_pk = secp
-        .recover_ecdsa(message, rec_sig)
+        .recover_ecdsa(
+            &secp256k1::Message::from_slice(message.as_bytes())?,
+            &secp256k1::ecdsa::RecoverableSignature::from_compact(
+                rec_sig.to_bytes().as_slice(),
+                secp256k1::ecdsa::RecoveryId::from_i32(rec_sig.v.to_byte() as i32).unwrap(),
```

Comment:
can we either handle the error case or propagate it here rather than unwrapping?

Reply:
addressed in [6d53b93](https://github.com/FuelLabs/fuel-block-committer/pull/222/commits/6d53b9346b90feb3e8f7a9f7a932f70478e09a74)